### ISSUE TITLE
Chapter 3: Exercise 3 using Titanic dataset

### ISSUE THREAD

**Issue by MiguelPCouto** [ageron/handson-ml2#153]
'Survived' should be dropped before fitting the model, therefore adding:
`train_df = train_df.drop('Survived', axis=1)` 
before Pipeline transformations. 

This would be very helpful for newbies.

**Comment by Ashish-Thanki** [comment]
That is correct, but you drop the data inplace you will need to set the survived column to a new variable (i.e. what you want to predict).

`y_train = train_data['Survived']`
